### PR TITLE
fix lint issues

### PR DESCRIPTION
**Pull request by NagaTulasi**

### CODE REVIEW

File: cmd/config.go [hunk 1]
```diff
@@ -32,7 +33,9 @@ import (
 )
 
 const (
-	ORDERED        = "ORDERED"
+	//ORDERED is exported constant
```
Comment (colin-axner) [inline cmd/config.go]:
```suggestion
	// ORDERED is exported channel type constant
```

File: cmd/config.go [hunk 2]
```diff
@@ -32,7 +33,9 @@ import (
...
+	ORDERED = "ORDERED"
+	//UNORDERED is exported constant
```
Comment (colin-axner) [inline cmd/config.go]:
```suggestion
	// UNORDERED is exported channel type constant
```

File: cmd/root.go [hunk 3]
```diff
@@ -25,6 +26,7 @@ import (
 	"github.com/spf13/viper"
 )
 
+//MB is ...
```
Comment (colin-axner) [inline cmd/root.go]:
```suggestion
// MB is a megabyte
```

File: go.mod [hunk 4]
```diff
@@ -3,26 +3,48 @@ module github.com/cosmos/relayer
...
 	github.com/Microsoft/go-winio v0.4.15 // indirect
 	github.com/avast/retry-go v2.6.0+incompatible
 	github.com/cenkalti/backoff/v3 v3.2.2 // indirect
 	github.com/containerd/continuity v0.0.0-20200928162600-f2cc35102c2a // indirect
 	github.com/cosmos/cosmos-sdk v0.41.0
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/gogo/protobuf v1.3.3
+	github.com/golangci/golangci-lint v1.36.0 // indirect
+	github.com/golangci/misspell v0.3.5 // indirect
+	github.com/golangci/revgrep v0.0.0-20180812185044-276a5c0a1039 // indirect
```
Comment (colin-axner) [inline go.mod]: Can you run `go mod tidy`?

File: relayer/channel-tx.go [hunk 5]
```diff
@@ -62,7 +62,8 @@ func (c *Chain) CreateOpenChannels(dst *Chain, maxRetries uint64, to time.Durati
 		// In the case of failure, increment the failures counter and exit if this is the 3rd failure
 		case !success:
 			failures++
-			c.Log(fmt.Sprintf("retrying transaction..."))
+			str := "retrying transaction..."
+			c.Log(str)
```
Comment (colin-axner) [inline relayer/channel-tx.go]:
```suggestion
			c.Log("retrying transaction...")
```
does this pass lint?

File: relayer/connection-tx.go [hunk 6]
```diff
@@ -58,7 +58,8 @@ func (c *Chain) CreateOpenConnections(dst *Chain, maxRetries uint64, to time.Dur
 		// increment the failures counter and exit if we used all retry attempts
 		case !success:
 			failed++
-			c.Log(fmt.Sprintf("retrying transaction..."))
+			str := ("retrying transaction...")
+			c.Log(fmt.Sprint(str))
```
Comment (colin-axner) [inline relayer/connection-tx.go]:
```suggestion
			c.Log("retrying transaction...")
```

File: relayer/controller.go [hunk 7]
```diff
@@ -4,13 +4,16 @@ import (
 	"encoding/json"
 )
 
+//SendToController is ...
```
Comment (colin-axner) [inline relayer/controller.go]:
```suggestion
// SendToController is ...
```

File: relayer/controller.go [hunk 8]
```diff
@@ -4,13 +4,16 @@ import (
...
 var SendToController func(needReply bool, str string) (string, error)
 
+//ControllerUpcall takes action interface type
```
Comment (colin-axner) [inline relayer/controller.go]:
```suggestion
// ControllerUpcall takes action interface type
```

File: relayer/encoding.go [hunk 9]
```diff
@@ -262,6 +262,7 @@ func (pc *ProtoCodec) UnpackAny(any *types.Any, iface interface{}) error {
 	return pc.interfaceRegistry.UnpackAny(any, iface)
 }
 
+//InterfaceRegistry returns protocodec struct type interfaceRegistry
```
Comment (colin-axner) [inline relayer/encoding.go]:
```suggestion
// InterfaceRegistry returns the ProtoCodec InterfaceRegistry
```

File: relayer/path.go [hunk 10]
```diff
@@ -92,6 +93,7 @@ func (p Paths) PathsFromChains(src, dst string) (Paths, error) {
 	return out, nil
 }
 
+//PathAction is ...
```
Comment (colin-axner) [inline relayer/path.go]:
```suggestion
// PathAction is ...
```

File: relayer/pathEnd.go [hunk 11]
```diff
@@ -13,6 +13,7 @@ import (
 var (
 	defaultChainPrefix = commitmenttypes.NewMerklePrefix([]byte("ibc"))
 	defaultDelayPeriod = uint64(0)
+	//DefaultUpgradePath is ..
```
Comment (colin-axner) [inline relayer/pathEnd.go]:
```suggestion
	// DefaultUpgradePath is the default IBC upgrade path set for an on-chain light client
```

File: relayer/pathEnd.go [hunk 12]
```diff
@@ -40,12 +41,14 @@ func OrderFromString(order string) chantypes.Order {
 	}
 }
 
+//GetOrder is ...
```
Comment (colin-axner) [inline relayer/pathEnd.go]:
```suggestion
// GetOrder returns the channel order for the path end
```

File: relayer/pathEnd.go [hunk 13]
```diff
@@ -40,12 +41,14 @@ func OrderFromString(order string) chantypes.Order {
...
 func (pe *PathEnd) GetOrder() chantypes.Order {
 	return OrderFromString(strings.ToUpper(pe.Order))
 }
 
 var marshalledChains = map[PathEnd]*Chain{}
 
+//MarshalChain is PathEnd
```
Comment (colin-axner) [inline relayer/pathEnd.go]:
```suggestion
// MarshalChain is PathEnd
```

File: relayer/pathEnd.go [hunk 14]
```diff
@@ -54,6 +57,7 @@ func MarshalChain(c *Chain) PathEnd {
 	return pe
 }
 
+//UnmarshalChain returns Marshalled chain
```
Comment (colin-axner) [inline relayer/pathEnd.go]:
```suggestion
// UnmarshalChain returns Marshalled chain
```

File: relayer/query.go [hunk 15]
```diff
@@ -402,10 +404,7 @@ func (c *Chain) QueryUnbondingPeriod() (time.Duration, error) {
 	return res.Params.UnbondingTime, nil
 }
 
-/////////////////////////////////////
-//          UPGRADES               //
-/////////////////////////////////////
-
+//QueryUpgradedClient returns upgraded client info
```
Comment (colin-axner) [inline relayer/query.go]:
```suggestion
// QueryUpgradedClient returns upgraded client info
```

File: relayer/query.go [hunk 16]
```diff
@@ -429,6 +429,7 @@ func (c *Chain) QueryUpgradedClient(height int64) (*codectypes.Any, []byte, clie
 	return client, proof, proofHeight, nil
 }
 
+//QueryUpgradedConsState returns upgraded consensus state and height of client
```
Comment (colin-axner) [inline relayer/query.go]:
```suggestion
// QueryUpgradedConsState returns upgraded consensus state and height of client
```

File: relayer/relayMsgs.go [hunk 17]
```diff
@@ -9,6 +9,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+//DeliverMsgsAction is struct
```
Comment (colin-axner) [inline relayer/relayMsgs.go]:
```suggestion
// DeliverMsgsAction is struct
```

**Review by colin-axner — COMMENTED** [review]
Fantastic! Left some nits/suggestions

File: relayer/channel-tx.go (hunk 5 above)
Comment (NagaTulasi) [reply relayer/channel-tx.go]: Yes, I've done this in latest commit

File: go.mod (hunk 4 above)
Comment (NagaTulasi) [reply go.mod]: yes, I've done in the latest commit

**Review by colin-axner — APPROVED** [review]
Fantastic, thank you @NagaTulasi!

**Pull request merged**